### PR TITLE
Move to auto image for copyright check

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: netci.groovy [hunk 1]
```diff
@@ -117,13 +117,13 @@ def CreateStyleCheckTasks = { taskString, taskName, checkName ->
...
 
         def newJob = job(jobName) {
-            label('ubuntu')
             steps {
                 shell(taskString)
             }
         }
 
         Utilities.simpleInnerLoopJobSetup(newJob, project, isPR, checkName)
+        Utilities.setMachineAffinity(newJob, 'Ubuntu14.04', 'latest-or-auto')
```
Comment (dilijev) [inline netci.groovy]: Why is the label used for latest-or-auto specifically 'Ubuntu14.04'?

**Comment by dilijev** [comment]
Otherwise LGTM

File: netci.groovy (hunk 1 above)
Comment (mmitche) [inline netci.groovy]: We used to specify just plain "ubuntu' but now there is 15.10 so we avoid confusion by eventually removing the original label and adding Ubuntu14.04.  The latest-or-auto bit specifies that the latest image should be used.  In this case it's a dynamically created VM.  Verified that the command works on those VMs

Comment (dilijev) [inline netci.groovy]: Sounds good.

Comment (dilijev) [inline netci.groovy]: By dynamically-created, do you mean that it is created when the job is queued, or less frequently than that?

Comment (mmitche) [inline netci.groovy]: Created as a need arises for the label.  It takes around 3-5 minutes for Linux VMs.  VMs will stick around for about an hour if idle before being deallocated, so there is a great change that during the day there will be no wait.

**Pull request merged**